.github/workflows/build-with-bal-test-graalvm.yml

```diff
@@ -17,5 +17,5 @@ jobs:
     secrets: inherit
     with:
       additional-build-flags: "-x test"
-      # Adding a test group for `liveServer` to disable the test as currently we don't have a live redshift-data instance 
-      additional-test-flags: "--groups liveServer"
+      # Adding a test group for `mockServer` to disable the tests, as we don't have a live redshift-data instance to run tests against 
+      additional-test-flags: "--groups mockServer"
```

Comment:
Do we have any mock tests here? If that's the case, what do we test?

Reply:
Nope we don't we need to skip the tests.

Reply:
In that case, I'd suggget using `--disable-groups liveServer`

Reply:
Fixed with [f41fd2f15c6b16e15056349bd536efbc82e8c7e0](https://github.com/ballerina-platform/module-ballerinax-aws.redshiftdata/pull/19/commits/f41fd2f15c6b16e15056349bd536efbc82e8c7e0)